sa/model.go

```diff
@@ -110,17 +110,22 @@ func SelectCertificates(s db.Selector, q string, args map[string]interface{}) ([
 	return models, err
 }
 
-func certStatusMetadataFields() []string {
+// CertStatusMetadataFields returns a slice of column names for rows in the
+// certificateStatus table.
+func CertStatusMetadataFields() []string {
```

Comment:
I'm not quite convinced that exporting this and using it in ocsp-updater is the right plan. For one thing, it means that any changes to this list of fields will require spooky action at a distance in order to avoid breaking ocsp-updater. Maybe ocsp-updater has its own list of fields it specifically cares about, completely independent from anything going on in the SA? I'm not sure either way yet.

Reply:
I mostly wanted to preserve the fact that this is what the code did before. I can agree that it's an odd pattern but I was trying to keep the size of this change minimal.